scriptmodules/libretrocores/lr-mame2010.sh

```diff
@@ -23,7 +23,7 @@ function build_lr-mame2010() {
     rpSwap on 750
     make clean
     local params=()
-    isPlatform "arm" && params+=("VRENDER=soft" "ARM_ENABLED=1")
+    isPlatform "arm" && params+=("VRENDER=soft" "ARM_ENABLED=1" "FORCE_DRC_C_BACKEND = 1")
```

Comment:
Should there be spaces around the = ?

Reply:
Actually it worked with the spaces so I didn't notice it, but you're right - it doesn't need them so I removed them for compliance. Thanks for spotting them.